index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html" charset="utf-8">
+	<link rel="StyleSheet" href="style.css" type="text/css">
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap&subset=cyrillic" rel="stylesheet">
+	<title>Турагенство PINK</title>
+  </head>
+  <body>
+    <header class="header">
+	  <nav class="header__nav nav">
+	    <div class="nav__menu-prewiew">
+	    <a class="nav__logo">
+	      <img class="nav__logo-image" src="" alt="Турагенство PINK" width="" height="">
+	    </a>
+	    <button class="nav__toggle" type="button">Меню</button>
+		</div>
+		<ul class="nav__list">
+		  <li class="nav__list-item"><a href="index.html" class="site-list__link">Главная</a></li>
+		  <li class="nav__list-item"><a href="photo.html" class="site-list__link">Фотографии</a></li>
+		  <li class="nav__list-item"><a href="form.html" class="site-list__link">Конкурс</a></li>
+		  <li class="nav__list-item"><a href="#" class="site-list__link">HTML Академия</a></li>
+		</ul>
+	  </nav>
+	</header>
+	<main class="main">
+	  <h1 class="visually-hidden">Турагенство PINK</h1>
+	  <img src="" alt="" width="" height="">
+	  <div class="page-main__app app">
+	    <h2 class="visually-hidden">Наше приложение</h2>
+	    <button type="button" class="app__button">Скачать приложение</button>
+	    <ul class="app__list">
+	      <li class="app__item"><a href="" class="app__link app__link--ios">IOS</a></li>
+		    <li class="app__item"><a href="" class="app__link app__link--android">Android</a></li>
+		    <li class="app__item"><a href="" class="app__link app__link--windows">Windows Phone</a></li>
+	    </ul>
+	    <p class="app__description">Доступно для iPhone, iPad, Android, Windows Phone, OS X, Windows 8</p>
+	  <section class="page-main__advantages advantages">
+	    <h2 class="visually-hidden">Наши преимущества</h2>
+	    <p class="advantages__offer">Устали от серости мегаполиса? Нам есть, что вам предложить!</p>
+	    <ul class="advantages__list">
+	      <li class="advantages__item">
+	        <h3 class="advantages__header">Поднимает настроение</h3>
```

Comment:
```suggestion
	        <h3 class="advantages__item-header">Поднимает настроение</h3>
```
Так как основной класс, у тебя называется advantages, что подразумевает множественное количество. 
То же самое касается и ```advantages__description```